### PR TITLE
[feat] added a card for empty wishlist/cart

### PR DESCRIPTION
**Pull request by aryan-13**
Added:
- A card to show that wishlist/cart is empty with a CTA

### CODE REVIEW

File: src/Components/CartComponents/CartComponents.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.empty-cart-card {
+	width: 100%;
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	/* background-color: var(--color-white); */
```
Comment (lannisterrr) [inline src/Components/CartComponents/CartComponents.css]: remove console.logs()

File: src/pages/CartPage.jsx [hunk 2]
```diff
@@ -13,26 +14,25 @@ function CartPage() {
 			<Navbar />
 
 			<section className="cart">
-				{/* <!-- HEADING --> */}
 				<h3 className="cart-heading heading-4 bold-2 centered">
 					{' '}
```
Comment (lannisterrr) [inline src/pages/CartPage.jsx]: remove {'' "}

**Review by lannisterrr — APPROVED** [review]
Good code structure , some minor suggesstions made.

**Review by anshikaa12 — APPROVED** [review]
clean code

**Pull request merged**